### ISSUE TITLE
Cutscene playback crash

### ISSUE THREAD

**Issue by scotsdez** [shinyquagsire23/OpenJKDF2#103]
On 0.6.8, I encountered a crash after manually playing back a cutscene (in this case, 0) and pressing Esc - I briefly saw the cutscene shrink in aspect ratio to 4:3, followed by a crash to desktop a few seconds later.

From the RPT file:
```
-------------------

Error occurred on Saturday, September 24, 2022 at 13:51:58.

openjkdf2-64.exe caused an Access Violation at location 00007FF7749FA0C1 in module openjkdf2-64.exe Reading from location 0000000000000000.

AddrPC           Params
00007FF7749FA0C1 000001F5FF01D3B0 000001F5FF2F9B01 FFFFFFFF00000001  openjkdf2-64.exe!jkGuiEsc_Show
00007FF774A1F73C 0000000000000006 00007FF700000004 0000000000000000  openjkdf2-64.exe!jkMain_EscapeMenuShow
00007FF774A1F6A4 0000000000000000 00007FF70000000F 0000000000000000  openjkdf2-64.exe!jkMain_GuiAdvance
00007FF774A3AC75 0000000000000000 000000D60000000F 0000000000000000  openjkdf2-64.exe!Window_Main_Loop
00007FF774A3AF66 0000000000000001 000001F5F4907BD0 000000D6FDBFF5D0  openjkdf2-64.exe!Window_Main_Linux
00007FF774A60CF2 0000000000000001 000001F5F4907BD0 000000000000003F  openjkdf2-64.exe!SDL_main
00007FF774A7B9F6 0000000000000000 0000000000000040 00007FF775224AE8  openjkdf2-64.exe!main_getcmdline
00007FF7749713AE 0000000000000000 0000000000000000 0000000000000000  openjkdf2-64.exe!__tmainCRTStartup
00007FF7749714C6 0000000000000000 0000000000000000 0000000000000000  openjkdf2-64.exe!WinMainCRTStartup
00007FFEC6FD7034 0000000000000000 0000000000000000 0000000000000000  KERNEL32.DLL!BaseThreadInitThunk
00007FFEC87C26A1 0000000000000000 0000000000000000 0000000000000000  ntdll.dll!RtlUserThreadStart

openjkdf2-64.exe
ntdll.dll   	6.2.19041.1949
KERNEL32.DLL	6.2.19041.1889
KERNELBASE.dll	6.2.19041.1949
apphelp.dll 	6.2.19041.1320
ADVAPI32.dll	6.2.19041.1682
msvcrt.dll  	7.0.19041.546
sechost.dll 	6.2.19041.1865
RPCRT4.dll  	6.2.19041.1806
GDI32.dll   	6.2.19041.1620
win32u.dll  	6.2.19041.2006
gdi32full.dll	6.2.19041.2006
msvcp_win.dll	6.2.19041.789
ucrtbase.dll	6.2.19041.789
USER32.dll  	6.2.19041.1503
IMM32.dll   	6.2.19041.546
ole32.dll   	6.2.19041.1202
combase.dll 	6.2.19041.1949
OLEAUT32.dll	6.2.19041.985
SETUPAPI.dll	6.2.19041.1741
cfgmgr32.dll	6.2.19041.1620
bcrypt.dll  	6.2.19041.1023
SHELL32.dll 	6.2.19041.1949
WS2_32.dll  	6.2.19041.546
VERSION.dll 	6.2.19041.546
OPENGL32.dll	6.2.19041.1806
glew32.dll  	2.1.0.0
GLU32.dll   	6.2.19041.1806
OpenAL32.dll	1.21.1.0
WINMM.dll   	6.2.19041.546
libGameNetworkingSockets.dll
IPHLPAPI.DLL	6.2.19041.1865
libwinpthread-1.dll	1.0.0.0
libgcc_s_seh-1.dll
libprotobuf.dll
dbghelp.dll 	10.0.19041.685
libstdc++-6.dll
shcore.dll  	6.2.19041.1865
exchndl.dll 	0.9.3.0
PSAPI.DLL   	6.2.19041.546
mgwhelp.dll 	0.9.3.0
dbgcore.DLL 	10.0.19041.685
uxtheme.dll 	6.2.19041.1620
IconCodecService.dll	6.2.19041.1
WindowsCodecs.dll	6.2.19041.1706
mscms.dll   	6.2.19041.746
USERENV.dll 	6.2.19041.572
ColorAdapterClient.dll	6.2.19041.546
icm32.dll   	6.2.19041.546
kernel.appcore.dll	6.2.19041.546
bcryptPrimitives.dll	6.2.19041.1415
clbcatq.dll 	2001.12.10941.16384
MMDevApi.dll	6.2.19041.1503
DEVOBJ.dll  	6.2.19041.1620
avrt.dll    	6.2.19041.546
hid.dll     	6.2.19041.546
WINTRUST.dll	6.2.19041.1949
CRYPT32.dll 	6.2.19041.1889
MSASN1.dll  	6.2.19041.546
dinput8.dll 	6.2.19041.1
inputhost.dll	6.2.19041.1741
wintypes.dll	6.2.19041.1949
CoreMessaging.dll	6.2.19041.746
CoreUIComponents.dll	6.2.19041.546
PROPSYS.dll 	7.0.19041.1741
ntmarta.dll 	6.2.19041.546
Windows.Gaming.Input.dll	6.2.19041.1566
twinapi.appcore.dll	6.2.19041.1865
MSCTF.dll   	6.2.19041.1949
AppXDeploymentClient.dll	6.2.19041.1865
nvoglv64.dll	31.0.15.1694
WTSAPI32.dll	6.2.19041.546
CRYPTBASE.DLL	6.2.19041.546
cryptnet.dll	6.2.19041.906
drvstore.dll	6.2.19041.1949
wldp.dll    	6.2.19041.1949
Windows.System.UserDeviceAssociation.dll	6.2.19041.746
usermgrproxy.dll	6.2.19041.1741
windows.storage.dll	6.2.19041.1949
shlwapi.dll 	6.2.19041.1706
dxcore.dll  	6.2.19041.546
nvspcap64.dll	3.25.1.27
profapi.dll 	6.2.19041.844
dwmapi.dll  	6.2.19041.746
powrprof.dll	6.2.19041.546
UMPDC.dll
WINSTA.dll  	6.2.19041.546
textinputframework.dll	6.2.19041.1806
AUDIOSES.DLL	6.2.19041.1865
resourcepolicyclient.dll	6.2.19041.546
XInput1_4.dll	6.2.19041.844
deviceaccess.dll	6.2.19041.1865
Ninput.dll  	6.2.19041.546
symsrv.dll  	10.0.19041.685
wininet.dll 	11.0.19041.1566
iertutil.dll	11.0.19041.1949
SspiCli.dll 	6.2.19041.1586
ondemandconnroutehelper.dll	6.2.19041.546
winhttp.dll 	6.2.19041.1682
mswsock.dll 	6.2.19041.546
NSI.dll     	6.2.19041.610
WINNSI.DLL  	6.2.19041.546
urlmon.dll  	11.0.19041.1949
srvcli.dll  	6.2.19041.1645
netutils.dll	6.2.19041.546
DNSAPI.dll  	6.2.19041.1865
rasadhlp.dll	6.2.19041.546
fwpuclnt.dll	6.2.19041.2006
schannel.DLL	6.2.19041.2006
mskeyprotect.dll	6.2.19041.1202
NTASN1.dll  	6.2.19041.546
DPAPI.DLL   	6.2.19041.546
CRYPTSP.dll 	6.2.19041.546
rsaenh.dll  	6.2.19041.1052
ncrypt.dll  	6.2.19041.662
ncryptsslp.dll	6.2.19041.546

Windows 6.2.9200
DrMingw 0.9.3
```

**Comment by scotsdez** [comment]
Further information:
This issue only seems to be present after starting a game (e.g. through the debug menu), aborting the mission, and then attempting to play a cutscene.
The intro cutscene played normally on starting the game.

**Comment by shinyquagsire23** [comment]
Recreated and fixed (7b74c75fdb9fd9a1ee08d3575be0fbd45f29ba3d), will be in the next release

**Comment by shinyquagsire23** [comment]
Fixed v0.6.9
